cmds/live.c

```diff
@@ -154,16 +156,13 @@ int command_live(int argc, char *argv[], struct opts *opts)
 
 		pr_dbg("live-record finished.. \n");
 		if (opts->report) {
-			pr_out("#\n# uftrace report\n#\n");
+			pr_dbg("start live-reporting...\n");
 			ret2 = command_report(argc, argv, opts);
-			if (ret == UFTRACE_EXIT_SUCCESS)
-				ret = ret2;
-
-			pr_out("\n#\n# uftrace replay\n#\n");
 		}
-
-		pr_dbg("start live-replaying...\n");
-		ret2 = command_replay(argc, argv, opts);
+		else {
+			pr_dbg("start live-replaying...\n");
+			ret2 = command_replay(argc, argv, opts);
+		}
```

Comment:
This part does not belong to this change, please remove.

Reply:
Right, it was a mistake. I just updated the commit. Thanks!